src/appleseed/renderer/modeling/light/maxomnilight.cpp

```diff
@@ -125,6 +142,7 @@ namespace
             outgoing = sample_sphere_uniform(s);
             value = m_values.m_intensity;
             probability = RcpFourPi<float>();
+            value *= probability;
```

Comment:
`value` is not supposed to be multiplied by probability here, it's the reason why we return both of them separately.